database/migrations/default/1744826895218_ems_remove_match_status_constraint/up.sql

```diff
@@ -23,25 +54,25 @@ BEGIN
     FROM people_list_view 
     WHERE id = NEW.person_id;
     
-    -- Update crah_pk of related EMS if not already matched to a crash
+    -- Update crash_pk of related EMS if not already matched to a person
     UPDATE ems__incidents 
     SET 
         crash_pk = matching_person_record.crash_pk,
         updated_by = NEW.updated_by,
         crash_match_status = 'matched_by_manual_qa'
     WHERE
-        incident_number = NEW.incident_number
+        id != NEW.person_id
+        AND incident_number = NEW.incident_number
         AND crash_pk is distinct from matching_person_record.crash_pk
         AND person_id IS NULL;
-
-    RETURN NEW;
+    RETURN null;
 END;
 $$ language plpgsql;
 
-comment on function update_person_crash_ems_match is 'Sets the ems__incident crash_pk based on the provided person_id and updates the crash_pk of all EMS records that share the same incident number';
+comment on function ems_update_incident_crash_pkis 'Sets the ems__incident crash_pk based on the provided person_id and updates the crash_pk of all EMS records that share the same incident number';
 
-create or replace trigger ems_update_person_match_trigger
+create or replace trigger ems_update_incident_crash_pk_trigger
 after update on ems__incidents
```

Comment:
so this trigger `ems_update_incident_crash_pk_trigger` runs after the `ems_update_person_crash_id_trigger`, maybe include in the comments that that is important. "Must run after the `ems_update_person_crash_id` trigger"

Reply:
👍 https://github.com/cityofaustin/vision-zero/pull/1777/commits/5d61a9f867a033f47f9c9e91795733f44705dc57